src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -15,15 +15,21 @@
 
 import copy
 import os
+import errno
 import shutil
 
+import time
+import subprocess
+
```

Comment:
[Imports need to follow the style guide](http://google.github.io/styleguide/pyguide.html#22-imports)
(ie: remove line 20 and sort imports).